fastlane/Fastfile

```diff
@@ -93,7 +93,11 @@ platform :ios do
   desc 'Module Pre-Release checks'
   lane :release do |options|
 
-    release_tag = version_get_podspec(path: ENV['REM_FL_PODSPEC_FILE'])
+    if File.exist?('../Podfile')
```

Comment:
why do we need this check? if it only supports `SPM`.

Reply:
ok, understood. It is for [ios-buildconfig](https://github.com/rakutentech/ios-buildconfig)

Reply:
earlier the release _tag was read from podspec. If there is no pod support (like RMCSDK) then the release_tag would be read from module which carries the release tag version that was provided at the time of workflow.